third_party/import-web/src/app.d.ts

```diff
@@ -0,0 +1,12 @@
+// See https://kit.svelte.dev/docs/types#app
+// for information about these interfaces
+declare global {
+	namespace App {
+		// interface Error {}
+		// interface Locals {}
+		// interface PageData {}
+		// interface Platform {}
```

Comment:
Remove if not needed

Reply:
boilerplate code from svelte-kit, let's not remove it